### DISCUSSION TITLE
Is it possible to create a window with webview? Or do I need to embed webview into a window I create myself?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [MicrosoftEdge/WebView2Feedback#3456] [Q&A] [answered]
Im making a PWA, trying to get rid of the artifact described at https://github.com/MicrosoftEdge/WebView2Feedback/issues/906 when resizing the window. Thinking that if WebView controls the window on its own, the artifact would probably disappear, is it possible to create an entire window with webview2?

**Comment by champnic** [comment, marked as answer]
Hey @ItsCubeTime - The WebView2 is a control that's typically placed within another window that you, the app, owns and controls. The exception is when opening a popup window with the "window.open" web API. You could try that, though I'm not sure it will avoid the visual artifacts or result in the experience you are trying to achieve.
